providers/implementations/rands/seed_src_jitter.c

```diff
@@ -194,20 +194,20 @@ static int jitter_generate(void *vseed, unsigned char *out, size_t outlen,
         return 0;
     }
 
-    if (adin != NULL && adin_len > 0) {
-        if (!ossl_rand_pool_add(pool, adin, adin_len, 0)) {
-            ERR_raise(ERR_LIB_PROV, ERR_R_RAND_LIB);
-            ossl_rand_pool_free(pool);
-            return 0;
-        }
-    }
-
     /* Get entropy from jitter entropy library. */
     entropy_available = ossl_prov_acquire_entropy_from_jitter(s, pool);
 
     if (entropy_available > 0)
         memcpy(out, ossl_rand_pool_buffer(pool), ossl_rand_pool_length(pool));
```

Comment:
the current code is correct, however I would still prefer to use something like: `MIN(ossl_rand_pool_length(pool), outlen))` so it's clear at the first glance without taking a tour through the code.

just a comment, I don't insist on this change here.

Reply:
This is not being changed and I do not think adding a MIN would be necessary. Perhaps adding a comment would make sense.